### PR TITLE
write env files to put options into environment

### PR DESCRIPTION
**Pull request by twobraids**
be able to write out environment variables for use in the shell.  automatically change '.' into '__'

### CODE REVIEW

**Comment by peterbe** [comment]
there is a bunch of commented out debugging(?). Wanna remove that so we don't forget later?

File: configman/value_sources/for_mapping.py [hunk 1]
```diff
@@ -40,3 +43,67 @@ def get_values(self, config_manager, ignore_mismatches, obj_hook=DotDict):
...
+    @staticmethod
+    def write(source_mapping, output_stream=sys.stdout):
+        ValueSource._write_ini(source_mapping, output_stream=output_stream)
+
+    #--------------------------------------------------------------------------
+    @staticmethod
+    def _namespace_reference_value_from_sort(key_value_tuple):
+        key, value = key_value_tuple
+        if value._reference_value_from:
+            return 'aaaaaa' + key
```
Comment (peterbe) [inline configman/value_sources/for_mapping.py]: This looks weird and deserves a comment to explain the reasoning.

**Comment by twobraids** [comment]
heh, I didn't even look at the code.  It was from an old branch that I never merged as I didn't think it important. I'll clean out those comments...

**Comment by twobraids** [comment]
selfie r+

**Pull request merged**